### ISSUE TITLE
Parsing error in class attributes doesn't report the line number

### ISSUE THREAD

**Issue by rdeltour** [IDPF/epubcheck#661]
Reported by Keith Snyder on [Twitter](https://twitter.com/noteon/status/674640501799305220), e.g. due to a missing end quote:

```xml
<p class="x>blah blah</p>
```

the error will be "Error while parsing file: the value of attribute class etc."

This is likely reported by Jing, see why the line number is not reported and if it can be fixed.

**Comment by tofi86** [comment]
Clearing assignee and removing from 4.1.0 milestone. Feel free to work on it and assign it back to 4.1.0 as long as we're still working on it.
